tests/test_compute_funs.py

```diff
@@ -1292,7 +1292,8 @@ def test_compute_everything():
                     np.testing.assert_allclose(
                         actual=this_branch_data[p][name],
                         desired=master_data[p][name],
-                        atol=1e-12,
+                        atol=1e-10,
```

Comment:
This is needed due to minor differences caused by CPU types. Previously pretty much all the gh runners used intel xeon cpus, but recently they've upgraded some to the newer AMD Ryzen ones. This seems to lead to tiny differences in some computed quantities. For most things the difference is on the order of machine precision so no problem, but for ``current_r`` at the axis the difference is ~1e-11. The difference in ``I_r`` is ~1e-17, but when we scale by mu0 we get larger differences for ``current_r``